core/src/main/scripts/importer/validateData.py

```diff
@@ -1275,14 +1274,24 @@ def checkMatchedNormSampleBarcode(self, value):
     def checkVerificationStatus(self, value):
         # if value is not blank, then it should be one of these:
         if self.checkNotBlank(value) and value.lower() not in ('verified', 'unknown', 'na'):
-            return False
+            self.logger.warning(
+                "Value in 'Verification_Status' not in MAF format",
```

Comment:
not used in frontend for any special cases, and simply ingested in backend, so OK. :+1: